usr/src/cmd/zdb/zdb.c

```diff
@@ -1566,7 +1567,8 @@ dump_deadlist(dsl_deadlist_t *dl)
 	    dle = AVL_NEXT(&dl->dl_tree, dle)) {
 		if (dump_opt['d'] >= 5) {
 			char buf[128];
-			(void) snprintf(buf, sizeof (buf), "mintxg %llu -> ",
+			(void) snprintf(buf, sizeof (buf),
+			    "mintxg %llu -> obj %llu",
```

Comment:
I'm amazed there isn't a gcc format warning for this... unless we ignore those in zdb.